distributions/raspijamulus.sh

```diff
@@ -22,10 +22,10 @@ fi
 if [ -d "${OPUS}" ]; then
   echo "The Opus directory is present, we assume it is compiled and ready to use. If not, delete the opus directory and call this script again."
 else
-  wget https://archive.mozilla.org/pub/opus/${OPUS}.tar.gz
-  tar -xzf ${OPUS}.tar.gz
-  rm ${OPUS}.tar.gz
-  cd ${OPUS}
+  wget https://archive.mozilla.org/pub/opus/"${OPUS}".tar.gz
+  tar -xzf "${OPUS}".tar.gz
```

Comment:
Style note...  I'd prefer to see the quoting around the _entire_ parameter, so here `"${OPUS}.tar.gz"` and later `"-j${NCORES}"`.

Reply:
I'm not sure if this is valid (I can't really test it either since my laptop is intel based).

Reply:
If `-j3` is valid and `NCORES=3` then `"-j${NCORES}"` better be identical to `-j3` or there's a bug in the shell.